libs/sdk-ui-dashboard/src/presentation/layout/DefaultDashboardLayoutRenderer/DashboardLayoutSection.tsx

```diff
@@ -54,8 +55,15 @@ export function DashboardLayoutSection<TWidget>(props: IDashboardLayoutSectionPr
                 screen={screen}
             />
         ));
+
         return gridRowRenderer
-            ? gridRowRenderer({ children: rowItems, screen, section, items: itemsInRow })
+            ? gridRowRenderer({
+                  children: rowItems,
+                  screen,
+                  section,
+                  items: itemsInRow,
+                  gridRowKey: uniqueId(),
```

Comment:
Is the gridRowKey persisted across re-renders? (if not, then what is its purpose?)

Reply:
I dropped the commit that includes this.